### DISCUSSION TITLE
Monitor community health in discussions with metrics, dashboard, data export

### DISCUSSION THREAD

**Discussion by aeneasr** [community/community#2931] [Discussions]
Monitoring community health (number of active contributors, website traffic, traffic sources) is currently lacking completely from discussions, but an important part of monitoring the community health. Is it planned to add a metrics dashboard and data export for monitoring, for example, number of discussions, comments, questions answered, unique users, and so on?

**Comment by marythought** [comment]
Hi, I am also interested in this and curious what metrics people are using to assess the health of their developer communities through discussion forums.

**Reply by michealabroy-wq** [reply]
People often look at metrics like:

* Number of active users
* Number of new posts and replies
* Response time to questions
* User engagement (likes, upvotes)
* Retention of contributors

These help show how active, helpful, and sustainable the community is.

**Comment by kamranikramofficial** [comment]
Yeah, monitoring community health is a really important aspect that's often overlooked. Having visibility into things like the number of active contributors, website traffic, engagement levels, and participation trends gives a much clearer picture of how the community is actually doing—not just anecdotally, but with real data.

As for whether it's planned to add a metrics dashboard and export features—ideally, yes, that should definitely be on the roadmap. A dashboard that tracks things like number of discussions, comments, answered questions, unique users, returning visitors, etc., would not only help moderators and admins make informed decisions, but also give the broader community transparency into its own growth and activity.

If it's not already in the works, it's definitely worth bringing up with the platform's team or including in feature requests. Community insights like these are essential for sustainable growth and making sure contributors feel seen and valued.

**Comment by ArpanJana2002** [comment]
Given the current limitations, I'd suggest:

Advocate for this feature through GitHub's community discussions or feedback channels
Build a custom solution using the GitHub API for your immediate needs
Monitor GitHub's public roadmap for future announcements about discussion analytics

This is indeed a notable gap in GitHub's community management tools, and many organizations managing active discussion communities would benefit from these metrics.

**Comment by gramwilliam336-cmd** [comment]
**[Youcine](https://youcineapkpremium.com/):** Add metrics, dashboard, and export options to monitor community health (#2931).

**Comment by iotwhip** [comment]
GitHub has actually already addressed this.
You can find a dedicated Community Insights dashboard right inside your repository. It lets you monitor key stats on your discussions and overall community health.

[a comment by an automated account is not shown]
